examples/undo-tree/client.js

```diff
@@ -1,8 +1,10 @@
 import Render   from './app/plugins/render'
 import UndoTree from './app/undo-tree'
+import Debugger from 'microcosm-debugger'
```

Comment:
Baaaaby steps, but it didn't feel like it was necessary anymore. Check it out:

https://github.com/vigetlabs/microcosm-debugger

Help me make it better!